src/renderer/npm.ts

```diff
@@ -32,23 +33,44 @@ const isUnique = (item: any, idx: number, arr: Array<any>): boolean => {
   return arr.lastIndexOf(item) === idx;
 };
 
+/**
+ * Checks if yarn is installed by checking if a binary
+ * with that name can be found.
+ */
+export async function getIsYarnInstalled(
+  ignoreCache?: boolean,
+): Promise<boolean> {
+  if (isYarnInstalled !== null && !ignoreCache) return isYarnInstalled;
+
+  const command =
+    process.platform === 'win32' ? 'where.exe yarn' : 'which yarn';
+
+  try {
+    await exec(process.cwd(), command);
+    return (isYarnInstalled = true);
+  } catch (error) {
+    console.warn(`getIsYarnInstalled: "${command}" failed.`, error);
+    return (isYarnInstalled = false);
+  }
+}
+
 /**
  * Checks if npm is installed by checking if a binary
  * with that name can be found.
  */
 export async function getIsNpmInstalled(
```

Comment:
Instead of making a new function, we could also keep the existing one and pass in `npm` or `yarn` as an arg, since the logic for checking if a package is installed is the same for both.

Reply:
Well seems reasonable, done in 0eec84c3ea7a133dfb89b51220be272febb4fe81